public/js/event-views.js

```diff
@@ -373,12 +474,28 @@ views.DialogView = Backbone.Marionette.Layout.extend({
             joinCap: joinCap,
             roomId: this.options.event.getRoomId()
         });
+
         $("input[type=text]", scope).val("");
         $(".yt-error, .url-error, .join-cap-error", scope).hide();
         $(".error", scope).removeClass(".error");
         scope.modal('hide');
     },
 
+    sendFollowupEmail: function(jqevt) {
+        jqevt.preventDefault();
+
+        var eventTitle = this.options.event.get("title");
+
+        $.ajax({
+            type: 'POST',
+            url: "/followup/event/" + this.options.event.id + "/",
+            data: {eventTitle: eventTitle }
```

Comment:
The server knows ``event.get("title")`` already.. do we need to post it as data?